src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -28,183 +24,155 @@
     get_sql_servers_operations,
     get_sql_server_usages_operations,
     get_sql_subscription_usages_operations,
-    get_sql_virtual_network_rules_operations
+    get_sql_virtual_network_rules_operations,
+    # get_sql_recommended_elastic_pools_operations
```

Comment:
Intentionally commented out?

Reply:
It is not really used even in the old code, so I commented out.

Reply:
I deleted it